BAC0/scripts/Complete.py

```diff
@@ -61,9 +62,7 @@ class Stats_Mixin:
     def number_of_devices(self):
         if not self.discoveredDevices:
             return 0
-        s = []
-        [s.append(x) for x in self.discoveredDevices.items() if x[1] > 0]
-        return len(s)
+        return len([x for x in self.discoveredDevices.items() if x[1] > 0])
```

Comment:
not sure if I'm mis-understanding the original intent here, so please let me know if I transposed incorrectly

Reply:
I needed a number of device to fill a dashboard widget... "complete" is this thing I'm always asking myself if I'll keep it...